packages/grid/src/EditorContext.ts

```diff
@@ -1,10 +1,11 @@
 import { createContext, useContext } from "react";
 
 export interface EditorContext {
-  initialText?: string;
+  editorText: string;
```

Comment:
this would be considered a breaking change  in future ?

How will we handle breaking changes, once the grid gets adoption ?
Do we know if anyone is using this already ?